elf/cmdline.cc

```diff
@@ -447,7 +447,7 @@ std::vector<std::string> parse_nonpositional_args(Context<E> &ctx) {
                    << "\n  Supported emulations:\n   elf_x86_64\n   elf_i386\n"
                    << "   aarch64linux\n   armelf_linux_eabi\n   elf64lriscv\n"
                    << "   elf64briscv\n   elf32lriscv\n   elf32briscv\n"
-                   << "   elf64lppc\n  elf64_sparc\n";
+                   << "   elf64lppc\n   elf64_sparc";
```

Comment:
I guess you need a `\n` before elf64_sparc? It looks like GNU's `ld -V` shows one target per line.

Reply:
Yep, there is '\n' before `elf64_sparc`.

Before my revision:
```
mold 1.4.2 (302a9ed8d8cb6d54ca68a3b8f33c8efe297febf0; compatible with GNU ld)
  Supported emulations:
   elf_x86_64
   elf_i386
   aarch64linux
   armelf_linux_eabi
   elf64lriscv
   elf64briscv
   elf32lriscv
   elf32briscv
   elf64lppc
  elf64_sparc

```

After:
```
mold 1.4.2 (302a9ed8d8cb6d54ca68a3b8f33c8efe297febf0; compatible with GNU ld)
  Supported emulations:
   elf_x86_64
   elf_i386
   aarch64linux
   armelf_linux_eabi
   elf64lriscv
   elf64briscv
   elf32lriscv
   elf32briscv
   elf64lppc
   elf64_sparc
```

Reply:
Oh, sorry, I missed that one.